server/src/main/java/org/apache/druid/client/BrokerServerView.java

```diff
@@ -265,29 +281,67 @@ private QueryableDruidServer removeServer(DruidServer server)
     return clients.remove(server.getName());
   }
 
+  private void polledSegmentsFromCoordinator(final ImmutableSortedSet<SegmentWithOvershadowedStatus> segmentWithOvershadowedStatusSet) {
```

Comment:
This class does not know whether the segments were obtained via polling or otherwise. We just need to clarify what we intend to do with those segments. Please rename the other variables accordingly.

```suggestion
  private void addHandedOffSegmentsToTimeline(final ImmutableSortedSet<SegmentWithOvershadowedStatus> allPublishedSegments) {
```